src/features/commands.ts

```diff
@@ -577,6 +580,30 @@ To integrate it into your editor: https://prettier.io/docs/en/editors.html`,
       });
     },
   },
+  {
+    words: [`!learn`],
+    help: `Resources for Learning React`,
+    category: "React/Redux",
+    handleMessage: (msg) => {
+      msg.channel.send({
+        embed: {
+          title: "Learning React",
+          type: "rich",
+          description: `
+          The official (beta) React docs are the best resource for learning React:
+          https://beta.reactjs.org
+
+          The official React docs still teach classes for the examples, but the concepts are still valid:
+          https://reactjs.org/docs/getting-started.html
+
+          There's also a "React with Hooks" version of the React docs that uses hooks and function components for all examples:
```

Comment:
Same thing as the above comment.

> The official (beta) React docs are the best source for learning React. They focus on functional components and hooks:

Reply:
It's something I wasn't sure about. My original thought was to just alias `!hooks` as `!learn`, but wasn't sure if others agreed there's kinda no reason to explicitly highlight hooks anymore.

Reply:
Yeah I think that makes sense to me. I think hooks have been the de facto mechanism for long enough, especially when the command is like 99% the same content just arranged differently.